eduHub/course/src/main/kotlin/com/arsiu/eduhub/course/infrastructure/persistence/repositories/CourseRedisRepositoryImpl.kt

```diff
@@ -0,0 +1,63 @@
+package com.arsiu.eduhub.course.infrastructure.persistence.repositories
```

Comment:
mongo and redis are two separate infrastructure components and should be placed into separate packages

Reply:
`+`